src/gui-qt4/libs/seiscomp3/gui/datamodel/eventlistview.cpp

```diff
@@ -2875,6 +2875,7 @@ void EventListView::readFromDatabase(const Filter &filter) {
 	if ( _withOrigins ) numberOfSteps -= eventDiff*20;
 	if ( _withFocalMechanisms ) numberOfSteps -= eventDiff*20;
 
+	if (numberOfSteps == 0 ) numberOfSteps+=1;
```

Comment:
I am not sure why that should help. If both values are set to zero then the progress bar turns into a busy indicator according to the Qt documentation (https://doc.qt.io/archives/qt-4.8/qprogressbar.html). In my opinion it is an error that should be fixed at Gtk side and not here although I also think that it does not hurt to apply this fix.

Reply:
Ok false fix, indeed it seems there is several issues through Gtk under CentOS7.